src/Selenium2Library/keywords/_javascript.py

```diff
@@ -129,19 +129,55 @@ def get_alert_message(self):
         """
         return self._close_alert()
 
+    def read_alert_message(self):
```

Comment:
This keyword is redundant. 

Let's modify ```get_alert_message``` to have a ```dismiss``` argument that defaults to ```True``` (for backwards compatibility).